tests/ui/rfcs/rfc-2632-const-trait-impl/tilde_const_on_impl_bound.stderr

```diff
@@ -1,11 +1,12 @@
-error[E0015]: cannot call non-const fn `<T as Foo>::foo` in constant functions
-  --> $DIR/tilde_const_on_impl_bound.rs:14:16
+error[E0308]: mismatched types
+  --> $DIR/tilde_const_on_impl_bound.rs:14:9
    |
 LL |         self.0.foo()
-   |                ^^^^^
+   |         ^^^^^^^^^^^^ expected `host`, found `true`
    |
-   = note: calls in constant functions are limited to constant functions, tuple structs and tuple variants
+   = note: expected constant `host`
+              found constant `true`
```

Comment:
The internal name `host` leaks.

Reply:
This is a known issue, there's a FIXME somewhere for improving this but I haven't had time to work on it yet